samples/dictionary.sample.yml

```diff
@@ -0,0 +1,771 @@
+  stem: atamih-
+
+- category: VTA-2
+  FST:
+    lemma: wâsênamawêw
+  head:     wâsênamawêw
+  paradigm: VTA
+  POS:      V
+  key:      wâsênamawêw
+  senses:
+    - definition: s/he puts the lights on for s.o.
+      sources:    [CW]
+    - definition: s/he lights (it/him) for s.o.
+      sources:    [CW]
+  stem: wâsênamaw-
+
+- category: VTA-3
+  FST:
+    lemma: mâciswêw
+  head:     mâciswêw
+  paradigm: VTA
+  POS:      V
+  key:      mâciswêw
+  senses:
+    - definition: s/he cuts s.o. open
+      sources:    [CW]
+  stem: mâcisw-
+
+- category: VTA-4
+  FST:
+    lemma: kispêwâtêw
+  head:     kispêwâtêw
+  paradigm: VTA
+  POS:      V
+  key:      kispêwâtêw
+  senses:
+    - definition: s/he defends s.o., s/he is overprotective of s.o.
+      sources:    [CW, MD]
+  stem: kispêwât-
+
+- category: VTA-5
+  FST:
+    lemma: ay-itêw
+  head:     ay-itêw
+  paradigm: VTA
+  POS:      V
+  key:      ay-itêw
+  senses:
+    - definition: s/he says so to s.o, s/he says so of s.o.
+      sources:    [CW]
+  stem: ay-it-
+
+- category: VTI-1
+  FST:
+    lemma: kâhtinam
+  head:     kâhtinam
+  paradigm: VTI
+  POS:      V
+  key:      kâhtinam
+  senses:
+    - definition: s/he obtains s.t., s/he attains s.t., s/he procures s.t.
+      sources:    [CW]
+    - definition: s/he pushes s.t.
+      sources:    [CW]
+  stem: kâhtin-
+
+- category: VTI-2
+  FST:
+    lemma: nahêkâcihtâw
+  head:     nahêkâcihtâw
+  paradigm: VTI
+  POS:      V
+  key:      nahêkâcihtâw
+  senses:
+    - definition: s/he is sparing of s.t.
+      sources:    [CW]
+  stem: nahêkâcihtâ-
+
+- category: VTI-3
+  FST:
+    lemma: oscoscocasiw
+  head:     oscoscocasiw
+  paradigm: VTI
+  POS:      V
+  key:      oscoscocasiw
+  senses:
+    - definition: s/he coughs a little
+      sources:    [CW]
+    - definition: s/he coughs up a little of something
+      sources:    [CW]
+  stem: oscoscocasi-
+
+- category: NA-3
+  FST:
+    lemma: atim
+  head:     atim
+  paradigm: NA
+  POS:      N
+  key:      atim
+  senses:
+    - definition: dog
+      sources:    [CW, MD]
+    - definition: horse
+      sources:    [CW]
+    - definition: beast of burden
+      sources:    [CW]
+  stem: atimw-
+
+- category: NA-3
+  FST:
+    lemma: atâhk
+  head:     atâhk
+  paradigm: NA
+  POS:      N
+  key:      atâhk
+  senses:
+    - definition: star
+      sources:    [CW, MD]
+  stem: atâhkw-
+
+- category: NA-1
+  FST:
+    lemma: acâhkos
+  head:     acâhkos
+  paradigm: NA
+  POS:      N
+  key:      acâhkos
+  senses:
+    - definition: star, little star
+      sources:    [CW, MD]
+  stem: acâhkos-
+
+- category: VAI-1
+  FST:
+    lemma: nîmiw
+  head:     nîmiw
+  paradigm: VAI
+  POS:      V
+  key:      nîmiw
+  senses:
+    - definition: s/he dances
+      sources:    [CW]
+  stem: nîmi-
+
+# nîminâniwan does not (and should not) have a LEXC entry
+- entryType:   wordform
+  fstAnalysis: nîmiw+V+AI+Ind+X
+  head:        nîminâniwan
+  lemmaSlug:   nîmiw
+  senses:
```

Comment:
The current outward behavior has been considered a working solution, so we have thought of retaining that, even though the underlying code implementation may change, cf. _nîminâniwan_ -> _nîmiw_

<img width="826" alt="image" src="https://user-images.githubusercontent.com/15003407/120875296-61063700-c568-11eb-954b-81b2ab79289f.png">


First, the wordform and definition for the wordform are shown for this entry - but by itself it doesn't have an independent paradigm. Rather, this is achieve via the lemma, which is used to fetch the lexeme entry for which the lemma is the head, which is shown second below. Since it is theoretically possible that there can be more than one lexeme entries for which the head string is equal to the lemma, we need to attach the unique key to the correct lexeme entry (cf. e.g. _kimiwan_ that we have to disambiguate for noun vs. verb).